ProductsWebApi/ProductsWebApi.Tests/ValidatorTests/UserValidatorTest.cs

```diff
@@ -0,0 +1,52 @@
+﻿using AutoFixture;
+using Microsoft.AspNetCore.Identity;
+using ProductsWebApi.Models;
+using ProductsWebApi.Services;
+using ProductsWebApi.Validator;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+namespace ProductsWebApi.Tests.ValidatorTests
+{
+    public class UserValidatorTest
+    {
+
+        [Theory]
+        [InlineData("", "test")]
```

Comment:
you don't always have to use Theory.
Especially for this case, you are testing only one possibility. 
Theory should be used, if you wanna run the same test suite, for multiple inputs

This can be a Fact. and values can be generated using fixtures